From 6a14d49c24e7a1e87cc4ed7339f14958d01a8cb7 Mon Sep 17 00:00:00 2001
From: Mianhan Liu <liumh1@shanghaitech.edu.cn>
Date: Thu, 30 Sep 2021 23:19:25 +0800
Subject: [PATCH] net/bridge/br_nf_core.c: remove superfluous headers

br_nf_core.c hasn't use any macro or function declared in linux/in_route.h,
linux/inetdevice.h, linux/sysctl.h and net/route.h.
Thus, these files can be removed from br_nf_core.c safely without
affecting the compilation of the ./net module
---
 net/bridge/br_nf_core.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/net/bridge/br_nf_core.c b/net/bridge/br_nf_core.c
index 8c69f0c95..67440936c 100644
--- a/net/bridge/br_nf_core.c
+++ b/net/bridge/br_nf_core.c
@@ -12,13 +12,9 @@
 
 #include <linux/module.h>
 #include <linux/kernel.h>
-#include <linux/in_route.h>
-#include <linux/inetdevice.h>
-#include <net/route.h>
 
 #include "br_private.h"
 #ifdef CONFIG_SYSCTL
-#include <linux/sysctl.h>
 #endif
 
 static void fake_update_pmtu(struct dst_entry *dst, struct sock *sk,
-- 
2.25.1

